README.md

```diff
@@ -1,2 +1,22 @@
 # beprof
 Beam Profile Analysing Tools
+
+Library provides methods to work with Beam Profiles which are sets of points <br>
+(might be 2-D or 3-D also with extra metadata) sorted by one of coordinates.
+
+beprof is based on nparray class from numpy, and it provides <br>
+numerous tools for different computations and data analysis.
+
+# installation
+
+Current version (0.1.0) is not available on PyPi, although once a <br>
+stable version is ready it will be pushed to PyPi repo.<br><br>
+
+For now, installation can be done from this GIT repository, using <br>
+`~$ pip3 install git+https://github.com/matkrak/beprof.git@pip3` <br>
```

Comment:
It works with your repository (tested it). You can put there link to original repo now.